modules/adding-node-iso-flags.adoc

```diff
@@ -0,0 +1,78 @@
+// Module included in the following assemblies:
+//
+// *nodes/nodes/nodes-nodes-adding-node-iso.adoc
+
+[id="adding-node-iso-flags{context}"]
+= Adding a node with command flags
+
+You can add a single node to your cluster by using command flags to specify configurations for the new node.
+
+.Prerequisites
+
+* You have installed the OpenShift CLI (`oc`)
+* You have an active connection to your target cluster
+* You have a kubeconfig file available
+
+.Procedure
+
+. Generate the ISO image by running the following command. The MAC address must be specified using a command flag.
++
+[source,terminal]
+----
+$ oc adm node-image create --kubeconfig='<path_to_kube_config>' --mac-address=<mac_address>
+----
++
+--
+where:
+
+`<path_to_kube_config>`:: Specifies the path to the kubeconfig file.
+`<mac_address>`:: Specifies the MAC address of the node that is being added.
+--
++
+[TIP]
+====
+To see additional flags that can be used to configure your node, run the following command:
+[source,terminal]
+----
+$ oc adm node-image create --help
+----
+====
+
+. Boot the ISO image.
+
+. Track progress of the node creation by running the following command:
++
+[source,terminal]
+----
+$ oc adm node-image monitor --kubeconfig='<path_to_kube_config>' <ip_address>
+----
++
+where:
+
+`<path_to_kube_config>`:: Specifies the path to the kubeconfig file.
+`<ip_address>`:: Specifies a list of the IP addresses of the nodes that are being added.
+
+. Once the node is created, review the pending certificate signing request (CSR) by running the following command:
```

Comment:
Because of the thread with Manoj where it was determined that this functionality depends on reverse DNS entries for the nodes, I think I'll still remove this step, but add a note to the previous step that without reverse DNS the users will have to run `oc get csr` to get the CSRs manually.